### ISSUE TITLE
Unable to approve or report flip - Report (NaN left)

### ISSUE THREAD

**Issue by rioda-org** [idena-network/idena-web#307]
One user had sent a video of a problem community members experienced few times up untill now

https://user-images.githubusercontent.com/37978870/182154778-bab05f02-d66b-410a-a524-bf550750dba3.mp4

**Comment by rioda-org** [comment]
Same for user in desktop app
https://discord.com/channels/634481767352369162/634481933803585546/1002938641783455784

https://user-images.githubusercontent.com/37978870/182155551-949deb56-124e-4512-aa80-18ec6ad1d704.mp4

**Comment by mnz2000** [comment]
I had this same problem in last validation. First I was able to successfully mark the flips. I approved some, reported some, skipped some. About half-way through, the whole user-interface disappeared and the web-app contents went all white.

I checked my node (running on a VPS), and it had the kswapd0 process taking most CPU time. I assumed the node had stopped answering the web-app. Indeed, after a while my web-app came back on, with the "Synchronizing..." message.

Once synchronized, I was able to continue the long session, except with the same symptoms as OP, Approve/Report buttons didn't work, and Report said ("NaN left"). I was able to submit the results with the approved/reported flips that I had done before the sync.

I'm guessing the OP may have had a shorter sync problem right in the beginning of the long session. Hope this helps to debug the issue.

**Comment by hlolve** [comment]
> I had this same problem in last validation. First I was able to successfully mark the flips. I approved some, reported some, skipped some. About half-way through, the whole user-interface disappeared and the web-app contents went all white.
> 
> I checked my node (running on a VPS), and it had the kswapd0 process taking most CPU time. I assumed the node had stopped answering the web-app. Indeed, after a while my web-app came back on, with the "Synchronizing..." message.
> 
> Once synchronized, I was able to continue the long session, except with the same symptoms as OP, Approve/Report buttons didn't work, and Report said ("NaN left"). I was able to submit the results with the approved/reported flips that I had done before the sync.
> 
> I'm guessing the OP may have had a shorter sync problem right in the beginning of the long session. Hope this helps to debug the issue.

I think your VPS was low on RAM and started to use SWAP, and that can make to fail validation. What are your VPS specifications ? vCPU, RAM, Storage size and type ?

**Comment by mnz2000** [comment]
Yes, obviously my VPS was low on RAM. My point was not to debug my VPS but to provide a way to reproduce the problem in the web-app.

**Comment by pocoloko** [comment]
I had this happen in a related but not exactly the same way... I accidentally closed the tab in Firefox during 2nd part of long session of validation. Then I did a "reopen closed tab" and the tab returned but asked me to login, after logging in I could see all the flips and my previous selections from 1st part of long, but I cannot select approve/report and the reports show NaN, I can only go to last flip in list and submit blank answers

![image](https://user-images.githubusercontent.com/65197321/185749588-b70b2de7-b7cf-46bb-9646-a7cc832776dc.png)
